src/radio/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import find_packages, setup
+
+package_name = 'radio'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='seamate',
+    maintainer_email='jungmaxwell@gmail.com',
+    description='TODO: Package description',
+    license='Apache-2.0',
+    tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+            'server-tcp = radio.server_tcp:main',
+            'client-tcp = radio.client_tcp:main',
+            'server-udp = radio.server_udp:main',
+            'client-udp = radio.client_udp:main',
```

Comment:
Also, weird observation:
Apparently, colcon complains if you use hyphen (-) in the module name.
So for example, if you rename the file `src/radio/radio/server_tcp.py` to `src/radio/radio/server-tcp.py` and modify the above lines to `server-tcp = radio.server-tcp:main`, colcon will refuse to build the radio package.

Reply:
Might be good idea to file this as an issue